README.md

```diff
@@ -8,6 +8,7 @@ const method = new ValidatedMethod({
   name, // DDP method name
   mixins, // Method extensions
   validate, // argument validation
+  callOptions, // options passed to Meteor.apply
```

Comment:
Is this confusing? Should it just be called `applyOptions`? It's not a huge deal, but seems weird that all the docs are like "use callOptions to pass options to Meteor.apply".